clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/CreateAzureServerGroupAtomicOperation.groovy

```diff
@@ -153,6 +150,7 @@ class CreateAzureServerGroupAtomicOperation implements AtomicOperation<Map> {
         task.updateStatus(BASE_PHASE, "Cleanup any resources created as part of server group upsert")
         try {
           if (serverGroupName) description.credentials.computeClient.destroyServerGroup(resourceGroupName, serverGroupName)
+          if (appGatewayPoolID) description.credentials.networkClient.removeAppGatewayBAPforServerGroup(resourceGroupName, description.appGatewayName, description.name)
```

Comment:
Please use braces for the `if` blocks. It makes long lines like these easier to read.